api/app.js

```diff
@@ -63,7 +63,16 @@ app.use(['/location', '/locations'], locationRouter);
 app.use(['/locationservice', '/locationservices'], locationServiceRouter);
 app.use(['/customerPet', '/customerPets'], customerPetRouter);
 app.use('/groomerSearch', groomerSearchRouter);
-app.use(['appointment', '/appointments'], appointmentsRouter);
+app.use(
+  [
+    '/appointment',
+    '/appointments',
+    '*/appointments',
+    '*/appointment',
+    '*/appointments',
```

Comment:
*/appointments is here twice, once we get this fixed it should be ready to merge!